src/apps/loan-list/list/list.tsx

```diff
@@ -70,7 +68,7 @@ const List: FC<ListProps> = ({
 
   return (
     <>
-      <div className="dpl-list-buttons m-32">
+      <div className="dpl-list-buttons my-32">
```

Comment:
What is `my-*`?

Reply:
https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/58e4403db616cfb6c09213951a78e0a75ef44c72/src/stories/Library/spacing/spacing.scss#L57